swagger.json

```diff
@@ -1941,9 +1475,154 @@
                     "type": "integer"
                 },
                 "totalDistance": {
-                    "type": "number"
+                    "type": "number",
+                    "description": "in meters"
                 }
             }
         }
+    },
+    "parameters": {
+        "entityId": {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "integer"
+        },
+        "all": {
+            "name": "all",
+            "in": "query",
+            "description": "Can only be used by admin users to fetch all entities",
+            "type": "boolean"
+        },
+        "userId": {
+            "name": "userId",
+            "in": "query",
+            "description": "Standard users can use this only with their own _userId_",
+            "type": "integer"
+        },
+        "deviceId": {
+            "name": "deviceId",
+            "in": "query",
+            "description": "Standard users can use this only with _userId_s, they have access to",
+            "type": "integer"
+        },
+        "DeviceInBody": {
```

Comment:
OK. I'll remove them.

I'd included them to dispel any ambiguities. But, I guess it seems pretty clear since they are under the _parameters_ reference branch.